device_doctor/lib/src/utils.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:logging/logging.dart';
+
+final Logger logger = Logger('DeviceDoctor');
+
+void fail(String message) {
+  throw BuildFailedError(message);
+}
+
+class BuildFailedError extends Error {
+  BuildFailedError(this.message);
+
+  final String message;
+
+  @override
+  String toString() => message;
+}
+
+/// Result of a health check for a specific parameter.
+class HealthCheckResult {
```

Comment:
Does this class belong to a different file? health.dart maybe?

Reply:
Moved to health.dart instead.